todo_app/templates/index.html

```diff
@@ -7,11 +7,19 @@ <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
 
+  <form action="add-todo" method="POST">
+    <label for="name">New to do item: </label>
+<input type="text" name="title" required />
+    <input type="submit" value="Add to list" />
+ </form>
+
```

Comment:
The indentation could be tidied up here:

```suggestion
  <form action="add-todo" method="POST">
    <label for="name">New to do item: </label>
    <input type="text" name="title" required />
    <input type="submit" value="Add to list" />
  </form>

```